modules/cbMap/processmap/DuplicateRelations.php

```diff
@@ -82,7 +83,10 @@ private function convertMap2Array() {
 
 		$relativemodules = array();
 		foreach ($xml->relatedmodules->relatedmodule as $r) {
-			$relativemodules[ (string)$r->module ] = (string)$r->relation;
+			$relativemodules[ (string)$r->module ] = array(
+				'relation' => (string)$r->relation,
+				'condition' => (string)$r->condition,
```

Comment:
I think this will emit a php warning if the condition is not present in the XML. Please try with a map that does not have conditions and eliminate the warning if it is there.